plugins/typescript/src/templates/fetcher.ts

```diff
@@ -69,7 +69,7 @@ export async function ${camel(prefix)}Fetch<
   THeaders,
   TQueryParams,
   TPathParams
->): Promise<TData> {
```

Comment:
Removed return type to infer

Reply:
This will not work, `response.json()` is `any`

![image](https://user-images.githubusercontent.com/1761469/187490238-28f36720-5da9-429f-98a8-c3d4aeca22fd.png)


This can work when wrapped in react-query hooks because the type definition is duplicated but that's it

Reply:
Yeah you are right. I will check for a workaround. 👍

Reply:
Technically, `TData` can be a `Blob`, but never tried to upload/download image with react-query ^^

Reply:
Yes, TData can be a Blob but TS was not happy. I reverted return type and casting blob as TData.

Reply:
Yes, TData can be a Blob but TS was not happy. I reverted return type and casting blob as TData.